app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -23,10 +23,11 @@
 public class DBHelper extends SQLiteOpenHelper {
     public static final String DATABASE_NAME = "Catima.db";
     public static final int ORIGINAL_DATABASE_VERSION = 1;
-    public static final int DATABASE_VERSION = 16;
+    public static final int DATABASE_VERSION = 17;
 
-    // NB: changing this value requires a migration
+    // NB: changing thoose values requires a migration
```

Comment:
Small grammar thing :)

```suggestion
    // NB: changing these values requires a migration
```

Reply:
Also in f19b597a